tests/task/task_runner/test_standard_task_runner.py

```diff
@@ -121,11 +121,12 @@ def test_on_kill(self):
         runner = StandardTaskRunner(job1)
         runner.start()
 
-        # give the task some time to startup
-        time.sleep(3)
-
+        # Give the task some time to startup
+        time.sleep(10)
         runner.terminate()
 
+        # Wait some time for the result
+        time.sleep(40)
```

Comment:
Hmm... Are we sure we cannot do some loop waiting for result - even in an active loop?.

Reply:
I thought about that, but longer sleep seems better than try / except in test.